apps/ex_wire/lib/ex_wire/packet/capability/eth.ex

```diff
@@ -17,11 +20,20 @@ defmodule ExWire.Packet.Capability.Eth do
     ]
   }
 
-  @supported_versions Map.keys(@version_to_packet_types)
+  @supported_versions MapSet.to_list(
```

Comment:
what exactly is happening here? 
could this be a function that you're able to test instead of a module attribute?

Reply:
Just getting the intersection of the Capabilities that we support and the Capabilities that are configured. I could make it a function, but I wanted it to be calculated once at compile time instead of on each call.

Reply:
seems somewhat stretching a module attribute. IDK, I have never used them for something else then simple values. if you're willing to bake them in the module... you could also write them by hand?

Reply:
```
a = Map.keys(version_to_packet_types)
b = Config.caps()|> |> Enum.filter(fn cap -> cap.name == name end) |> Enum.map(fn cap -> cap.version end)
'>' = [62] = a -- (a -- b)
```

Reply:
after we discussed offline, I think it also makes sense to leave them as attributes.